subjects/DPxAI/transform-objects/README.md

```diff
@@ -0,0 +1,79 @@
+## Transform Objects
+
+> Mindful AI mode
+
+### Context
+
+Imagine your favorite robot friend with all its cool features: a type, weight, and operational status.
+
+In JavaScript, we use objects to group these properties together, making it easy to manage and tweak our robot’s settings. Let’s see how we can modify, add, or remove properties in a JavaScript object to make our robot even cooler!
+
+### AI-Powered Learning Techniques
+
+`Reflective Technique:`
+This type of prompt encourages the AI to help learners reflect on their understanding by asking questions and prompting them to think critically about the concepts.
+
+Find the examples across the subject ;)
+
+### Concepts
+
+### Modifying Objects
+
+Let's start with a robot object:
+
+```js
+const robot = {
+  points: 0,
+  code: "75lai78wn",
+};
+```
+
+### Adding a New Property
+
+Give your robot a name:
+
+```js
+robot.name = "RoboMax";
+```
+
+### Changing a Property Value
+
+Boost your robot’s points:
+
+```js
+robot.points = 10;
+```
+
+### Removing a Property
+
+Remove a property:
+
+```js
+robot.code = undefined;
+```
+
+#### **`Prompt Example`**:
+
+- "How does modifying an object's property differ from adding a new property or removing an existing one?"
+
+- "Can you think of a scenario where using an object to group related properties would be more beneficial than using separate variables?"
+
+### Instructions
+
+#### Task 1:
+
+Modify the provided robot variable:
```

Comment:
Is `robot` the variable name provided? If that's the case, I would add the formatting to make it clear

Reply:
Yes, it was modified.